src/libstd/env.rs

```diff
@@ -618,7 +618,7 @@ pub mod consts {
     #[stable(feature = "env", since = "1.0.0")]
     pub const ARCH: &'static str = super::arch::ARCH;
 
-    /// The family of the operating system. In this case, `unix`.
+    /// The family of the operating system.
```

Comment:
`of the operating system in use`

That would align this with the doc comment below, and then it's clear to which `operating system` this actually refers to.